### PR TITLE
Add customized sort query signature (#624)

### PR DESCRIPTION
**Pull request by YarLikviD**
Implementation Issue (#624)
I added request to the sort_query method. 
It would be possible to sort only by clicking on the column, but this would reduce the flexibility of customization.

### CODE REVIEW

File: sqladmin/models.py [hunk 1]
```diff
@@ -954,6 +943,20 @@ def get_export_columns(self) -> List[str]:
             defaults=self._list_prop_names,
         )
 
+    def _get_sort_fields(self, request: Request) -> List[Tuple[str, bool]]:
```
Comment (aminalaee) [inline sqladmin/models.py:946]: I think this method can be inlined in the `sort_query`, or at least be public to be used so overriding the sort_query is made easy. But I think it's small enough to fit into the public `sort_query`.

**Review by aminalaee — COMMENTED** [review]
Looks great, thanks for the PR, just a minor comment.

File: sqladmin/models.py (hunk 1 above)
Comment (YarLikviD) [reply sqladmin/models.py:946]: Do you mean something like that? Only one method.
```py
    def sort_query(self, stmt: Select, request: Request) -> Select:
        """
        A method that is called every time the fields are sorted
        and that can be customized.
        By default, sorting takes place by default fields.
        """
        sort_by = request.query_params.get("sortBy", None)
        sort = request.query_params.get("sort", "asc")

        if sort_by:
            sort_fields = [(sort_by, sort == "desc")]
        else:
            sort_fields = self._get_default_sort()

        for sort_field, is_desc in sort_fields:
            if is_desc:
                stmt = stmt.order_by(desc(sort_field))
            else:
                stmt = stmt.order_by(asc(sort_field))

        return stmt
```

**Review by aminalaee — APPROVED** [review]
Thanks for the PR :+1:

**Pull request merged**